src/types/realm_user_count.js

```diff
@@ -0,0 +1,20 @@
+/**
+ * A Tozny ID Realm belonging to an account.
+ */
+class RealmUserCount {
+  constructor(identity_count) {
+    this.identity_count = identity_count
```

Comment:
```suggestion
    this.identityCount = identity_count
```
😄 💜 

also would name this file `realmUserCount` 🤪 
🐫